packages/data-context/src/sources/migration/shouldShowSteps.ts

```diff
@@ -15,7 +15,7 @@ function getTestFilesGlobs (config: LegacyCypressConfigJson, type: 'component' |
     return ([] as string[]).concat(glob)
   }
 
-  return ['**/*.{js,ts,jsx,tsx,coffee}']
+  return ['**/*.{js,ts,jsx,tsx,coffee,cjsx}']
```

Comment:
Should move `**/*.{js,ts,jsx,tsx,coffee,cjsx}` to a constant, it's being used in a few places.